forge-game/src/main/java/forge/game/card/CardUtil.java

```diff
@@ -268,8 +268,8 @@ public static Card getLKICopy(final Card in, Map<Integer, Card> cachedMap) {
         newCopy.setMonstrous(in.isMonstrous());
         newCopy.setRenowned(in.isRenowned());
         newCopy.setSolved(in.isSolved());
-        newCopy.setSuspectedTimestamp(in.getSuspectedTimestamp());
         newCopy.setSuspected(in.isSuspected());
+        newCopy.setSuspectedTimestamp(in.getSuspectedTimestamp());
```

Comment:
@tool4ever why exactly did you do this?

the setSuspected() function explicit was looking to the timestamp to not mess up the created properties

it might not matter because they are overwritten later anyway, but like this, there is unnecessary keywords generated